statik/database.py

```diff
@@ -418,6 +418,14 @@ def __init__(self, model=None, session=None, **kwargs):
                     "Attempting to look up primary keys for ManyToMany " +
                     "field relationship: %s", self.field_values[field_name]
                 )
+
+                # check if non-string items are present
+                if not all(isinstance(item, str) for item in self.field_values[field_name]):
```

Comment:
this will fail on supported Python 2.7 if unicode is used, because `str` isnt `unicode` on Python 2.  Please create some tests, especially for unicode content.